src/Document/FeColorMatrixElement.ts

```
@@ -0,0 +1,154 @@
+					0.213 + 0.787 * s, 0.715 - 0.715 * s, 0.072 - 0.072 * s, 0, 0,
+					0.213 - 0.213 * s, 0.715 + 0.285 * s, 0.072 - 0.072 * s, 0, 0,
+					0.213 - 0.213 * s, 0.715 - 0.715 * s, 0.072 + 0.928 * s, 0, 0,
+					0, 0, 0, 1, 0,
+					0, 0, 0, 0, 1
+				];
+				break;
+			}
+
+			case 'hueRotate':
+
+				const a = matrix[0] * Math.PI / 180.0;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)